Migrate wiki (#123)
jenkinsci/workflow-durable-task-step-plugin: CHANGELOG.md

> ## Changelog

Reviewer: Do you want to pull everything up a level?
```suggestion
# Changelog
```

> ### 2.34

Reviewer: ```suggestion
## 2.34
```

And so on...

# Changelog

## 2.35
2019 Nov 01

> - Update durable-task (v1.33)

Reviewer 2: nit: I wouldn't put this line in the changelog, since it doesn't really mean anything to users by itself, and a good portion of users are probably already running it anyway. IMO, the second line covers the change that users might care about in relation to the durable-task update.

> - Add `pwsh` step to support powershell name change ([JENKINS-48803](https://issues.jenkins-ci.org/browse/JENKINS-48803))

Reviewer 2: I might reword like this:

```suggestion
- Add `pwsh` step, which works exactly like the existing `powershell` step, but runs `pwsh.exe` instead of `powershell.exe`, for use with PowerShell Core 6.0 and newer. ([JENKINS-48803](https://issues.jenkins-ci.org/browse/JENKINS-48803))
```
